README.md

```diff
@@ -13,16 +13,16 @@ Motivation
 
 Good Morning is intended to be used as an extension to [QSToolKit (QSTK)](http://wiki.quantsoftware.org/index.php?title=QuantSoftware_ToolKit) library. By using [QSTK](http://wiki.quantsoftware.org/index.php?title=QuantSoftware_ToolKit) you can easily download historical stock market data from [Yahoo Finance](http://finance.yahoo.com/). You can also download fundamental financial data from [Compustat](https://www.capitaliq.com/home/what-we-offer/information-you-need/financials-valuation/compustat-financials.aspx). However, most individuals and institutions do not have access to [Compustat](https://www.capitaliq.com/home/what-we-offer/information-you-need/financials-valuation/compustat-financials.aspx). Good Morning attempts to mitigate this limitation by providing a very simple Python interface for downloading fundamental financial data from [financials.morningstar.com](http://financials.morningstar.com/).
 
-Example
-=======
+Example of Downloading Key Ratios from MorningStar
+==================================================
 
-    import good_morning as gm
+    import morningstar as gm
```

Comment:
Yes, @petercerno, this code works.  This isn't a file, but it's  folder to hold a module.  The `__init__.py` file in the folder is what makes the code inside the folder importable.  

The CI test I had made sure it worked.  And, even without the CI test, if you install this code, and run the example in the updated README, it works. 

To give some info on the CI code, this is exactly what CI code is meant to support.  I runs automatic tests when someone adds/pushes code to github.